packages/api/src/brevo-contact/brevo-contacts.service.ts

```diff
@@ -53,12 +53,31 @@ export class BrevoContactsService {
         return created;
     }
 
+    public async createTestContact({
+        email,
+        attributes,
+        scope,
+    }: {
+        email: string;
+        attributes?: BrevoContactAttributesInterface;
+        scope: EmailCampaignScopeInterface;
+    }): Promise<boolean> {
+        const testTargetGroupForScope = await this.targetGroupService.createIfNotExistTestTargetGroupForScope(scope);
+
+        const brevoIds = [testTargetGroupForScope.brevoId];
```

Comment:
imo you do not need this variable, can simply pass the array like this in the brevo `createTestContact` function.